cinema/views.py

```diff
@@ -0,0 +1,37 @@
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+
+from cinema.models import Movie
+from cinema.serializer import MovieSerializer
+from rest_framework.response import Response
+from rest_framework import status
+
+
+@api_view(["GET", "POST"])
+def movies_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+    else:
+        serializer = MovieSerializer(data=request.data)
+        if serializer.is_valid():
+            if serializer.save():
+                return Response(serializer.data, status=status.HTTP_201_CREATED)
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
it is bad practice, use: serializer.is_valid(raise_exception=True)
(check it in checklist in Readme)
fix hear and below

Reply:
Valeria is right - fix as per checklist